File: Easy/feedBack/feedBack.c
#include <stdio.h>
#include <assert.h>
#include <string.h>

int main(void){
	int T,flag,len;
	char S[100000];

	scanf("%d",&T);
	assert(T>=1 && T<=100);

	while(T>0){
		scanf("%s",S);
		len = strlen(S);

		for(int i=0;i<len;i++){
			assert(S[i]=='0' || S[i]=='1');
		}

		flag = 0;

		if(len<3){
			printf("Bad\n");
		}else{
			for(int i=0;i<len-2;i++){
				if(S[i]=='0' && S[i+1]=='1' && S[i+2]=='0'){
					flag = 1;
					break;
				}else if(S[i]=='1' && S[i+1]=='0' && S[i+2]=='1'){
					flag = 1;
					break;
				}
			}

			if(flag==1){
				printf("Good\n");
			}else{
				printf("Bad\n");
			}
		}
		
		T--;
	}

	return 0;
}

File: Easy/feedBack/feedBack.java
//run as java -ea feedBack
import java.util.Scanner;

public class feedBack{
	public static void main(String[] args){
		int T;
		String S;
		Scanner scan = new Scanner(System.in);

		T = scan.nextInt();
		assert(T>=1 && T<=100) : "T must be 1 <= T <= 100";

		while(T>0){
			S = scan.next();
			assert(S.length()>=1 && S.length()<=100000) : "S must be length 1 <= |S| <= 10^5";

			for(int i=0;i<S.length();i++){
				assert(S.charAt(i)=='0' || S.charAt(i)=='1') : "S must only have 0 or 1";
			}

			if(S.contains("010") || S.contains("101")){
				System.out.println("Good");
			}else{
				System.out.println("Bad");
			}

			T--;
		}
	}
}